README.md

```diff
@@ -7,3 +7,25 @@ Arbor is a library for implementing performance portable network simulations of
 An installation guide and library documentation are available online at [Read the Docs](http://arbor.readthedocs.io/en/latest/).
 
 [Submit a ticket](https://github.com/arbor-sim/arbor) if you have any questions or want help.
+
+### Citing Arbor
+
+It is possible to cite specific versions of Arbor used to run a simulation via Zenodo:
+* Version 0.2: [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.2583709.svg)](https://doi.org/10.5281/zenodo.2583709)
+* Version 0.1: [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.1459679.svg)](https://doi.org/10.5281/zenodo.1459679)
+
+The following BibTeX can be used to cite Arbor:
+
+```
+@INPROCEEDINGS{
+    paper:arbor2019,
+    author={N. A. {Akar} and B. {Cumming} and V. {Karakasis} and A. {Küsters} and W. {Klijn} and A. {Peyser} and S. {Yates}},
+    booktitle={2019 27th Euromicro International Conference on Parallel, Distributed and Network-Based Processing (PDP)},
+    title={{Arbor -- A Morphologically-Detailed Neural Network Simulation Library for Contemporary High-Performance Computing Architectures}},
+    year={2019}, month={Feb}, volume={}, number={},
```

Comment:
`month` should be either `feb` (without the brackets) or `2` (this is for portability: month names like `feb` are essentially macros that expand to a locale-specific string.)